lib/ts/recipe/openid/utils.ts

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import NormalisedURLDomain from "../../normalisedURLDomain";
+import NormalisedURLPath from "../../normalisedURLPath";
+import { NormalisedAppinfo } from "../../types";
+import { APIInterface, RecipeInterface, TypeInput, TypeNormalisedInput } from "./types";
+
+export function validateAndNormaliseUserInput(appInfo: NormalisedAppinfo, config?: TypeInput): TypeNormalisedInput {
+    let issuerDomain = appInfo.apiDomain;
+    let issuerPath = appInfo.apiBasePath;
+
+    if (config !== undefined) {
+        issuerDomain = new NormalisedURLDomain(config.issuer);
+        issuerPath = new NormalisedURLPath(config.issuer);
+
+        if (!issuerPath.equals(appInfo.apiBasePath)) {
+            throw new Error("Issuer URL must end with apiBasePath");
```

Comment:
Change the message to "Issuer URL must end with apiBasePath. The default value is /auth"

Reply:
Done